### PR TITLE
feat(charts): Added Histogram

### PR DESCRIPTION
**Pull request by SanjeevLakhwani**

### CODE REVIEW

File: src/Components/Charts/BentoBarChart.tsx [hunk 1]
```diff
@@ -1,129 +1,16 @@
...
-      </p>
-    </div>
-  );
+    return (
+        <BaseBarChart chartFill={chartFill} otherFill={otherFill} {...params} />
+    );
 };
 
-export default BentoBarChart;
+export default BentoBarChart;
```
Comment (davidlougheed) [inline src/Components/Charts/BentoBarChart.tsx:16]: missing ending newline

File: src/Components/Charts/BentoHistogram.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {HistogramProps} from "../../types/chartTypes";
```
Comment (davidlougheed) [inline src/Components/Charts/BentoHistogram.tsx:2]: add consistent spacing inside `{}`

File: src/Components/Charts/BentoHistogram.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+import { useChartTheme } from '../../ChartConfigProvider';
```
Comment (davidlougheed) [inline src/Components/Charts/BentoHistogram.tsx:4]: does this have a shortened import form?

File: src/Components/Charts/BentoHistogram.tsx (hunk 2 above)
Comment (davidlougheed) [reply src/Components/Charts/BentoHistogram.tsx:2]: does this have a shortened import form?

File: src/Components/Charts/BentoHistogram.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import BaseBarChart from "./BaseBarChart";
+
+
+const BentoHistogram: React.FC<HistogramProps> = ({colorTheme = 'default', ...params}) => {
+      const { fill: chartFill, other:otherFill } = useChartTheme().histogram[colorTheme];
```
Comment (davidlougheed) [inline src/Components/Charts/BentoHistogram.tsx:9]: bad indent

File: src/Components/Charts/BentoBarChart.tsx [hunk 5]
```diff
@@ -1,129 +1,16 @@
...
-  LABEL_STYLE,
-  MAX_TICK_LABEL_CHARS,
-  TITLE_STYLE,
-  TICKS_SHOW_ALL_LABELS_BELOW,
-  UNITS_LABEL_OFFSET,
-  TICK_MARGIN,
-  COUNT_KEY,
-} from '../../constants/chartConstants';
+import React from 'react';
+import {BarChartProps} from "../../types/chartTypes";
```
Comment (davidlougheed) [inline src/Components/Charts/BentoBarChart.tsx:2]: consistent spacing

**Review by davidlougheed — CHANGES_REQUESTED** [review]

File: src/constants/chartConstants.ts [hunk 6]
```diff
@@ -42,6 +42,8 @@ export const COLORS: HexColor[] = [
   '#3B3EAC',
 ];
 
+export const NEW_CHART_COLORS: HexColor[] = ['#F94144', '#F3722C', '#F8961E', '#F9C74F', '#90BE6D', '#2D9CDB'];
```
Comment (davidlougheed) [inline src/constants/chartConstants.ts:45]: Regarding chart colours, I don't think there are enough here - many of our pie charts have a lot of sections, so we run into repeating colours pretty quickly here.

Comment (SanjeevLakhwani) [reply src/constants/chartConstants.ts:45]: okie I can try something

**Pull request merged**